pageserver/src/bin/pageserver.rs

```diff
@@ -574,6 +573,7 @@ fn start_pageserver(
         );
     }
 
+    startup_checkpoint("complete", "Startup complete");
```

Comment:
Note the future at 441...448 should most of the time in prod complete later than this, because we take like 10s to load and activate all tenants, whereas we come to this line as soon as all load work has been scheduled.

Reply:
Well, I guess the reaching here before loading the tenants could be a startup as well.

Reply:
Oops, I missed that all of the earlier work was happening in a background task too.  Reverted the commit that moved this "complete" line here.